app/src/main/AndroidManifest.xml

```diff
@@ -103,6 +103,7 @@
 
                 <data
                     android:host="www.kickstarter.com"
+                    android:pathPrefix="/discover"
```

Comment:
By adding this path, the open button on the main page website would not be recognized. Let's not add the discover path yet, until figuring out a more generic solution.